shared/src/main/java/uk/gov/di/authentication/shared/services/AuditService.java

```diff
@@ -64,7 +64,7 @@ public void submitAuditEvent(
         var txmaAuditEvent =
                 auditEventWithTime(event, () -> Date.from(clock.instant()))
                         .withClientId(clientId)
-                        .withComponentId(configurationService.getOidcApiBaseURL().orElse("UNKNOWN"))
+                        .withComponentId("AUTH")
```

Comment:
Shall we pull this string out into a constant? (I appreciate we haven't always been super consistent with this kind of thing so far :-D)